torch_xla/csrc/ir.h

```diff
@@ -59,7 +59,7 @@ struct Output {
   struct Hasher {
     size_t operator()(const Output& output) const {
       size_t h = reinterpret_cast<std::ptrdiff_t>(output.node);
-      return h ^ (h >> 11) ^ output.index;
+      return h ^ (output.index + 0x9e3779b97f4a7c15 + (h << 6) + (h >> 2));
```

Comment:
Any reason why we couldn't use `std::hash`?

Reply:
We are combining here.
This is the same "combine" code in utill.h, but cannot include util.h in this file for layering issues.